code/game/machinery/computer/communications.dm

```diff
@@ -278,7 +278,7 @@
 
 		// OMG CENTCOM LETTERHEAD
 		if("MessageCentCom")
-			if(authenticated==2)
+			if(authenticated==1)

```

Comment:
Seems like this disables it when logged in as Cap, probably just want `if(authenticated)` as elsewhere in the file

Reply:
I am the big dumb. Thanks for catching this.